src/App.php

```diff
@@ -691,47 +691,83 @@ public function stickyForget(string $name): void
      * @param bool                                     $useRequestUrl       Simply return $_SERVER['REQUEST_URI'] if needed
```

Comment:
what is the `$useRequestUrl` usecase?

it replaces `$page`, so the design does not seems to be right, I am inclined to remove it completely, is that possible and what usecases will it break?

Reply:
i think you are right, was there but apparently is not used anywhere in codebase.

Reply:
addressed in https://github.com/atk4/ui/pull/2107